package.json

```diff
@@ -21,6 +21,9 @@
   },
   "typesVersions": {
     "*": {
+      "types": [
```

Comment:
Question: isn't there supposed to be a version number here? Or does the `*` mean something like "support all versions of typescript"?

Reply:
that's correct yea -- this library hasn't declare _any_ typescript support, so it's kind of a gray area -- but also correct `*` means any / all versions of typescript